.pubnub.yml

```diff
@@ -1,8 +1,13 @@
 name: c-core
 schema: 1
-version: 2.12.4
+version: "2.12.5"
```

Comment:
Why quotes?

Reply:
@davidnub matter of habit, since this is how string version (per specs it should be string) appear in some other our SDKs. If important, I can remove double-quotes.

Reply:
Oh. I just noticed that it wasn't there before and wanted to make sure it didn't cause problems processing. If it works then I have no problems with it.